lib/src/configuration.cc

```diff
@@ -337,6 +338,25 @@ std::string Configuration::setupEventLogLogging(std::string loglevel)
     return loglevel;
 }
 
+std::string Configuration::setupSyslogLogging(std::string loglevel)
+{
+#ifdef _WIN32
+    throw Configuration::Error {
+      lth_loc::format("eventlog is available only on POSIX platforms") };
+#else  // _WIN32
+#ifdef HAS_LTH_SYSLOG
+    lth_log::setup_syslog_logging("pxp-agent");
```

Comment:
This initializes the logger with a garbled ID, such as `/ò*[10485804]` instead of `pxp-agent[10485804]`, and I don't understand why.

Reply:
https://stackoverflow.com/a/27627466/6648276

TIL